integration-tests/src/test/delete_authority.test.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiPromise, KeyringPair } from 'creditcoin-js';
+import { creditcoinApi } from 'creditcoin-js';
+import { testData } from './common';
+import { testIf } from '../utils';
+import { addAuthorityAsync } from 'creditcoin-js/lib/extrinsics/add-authority';
+import { POINT_01_CTC } from '../constants';
+
+const globals = (global as any);
+
+describe('DeleteAuthority', (): void => {
+    let api: ApiPromise;
+    let sudoSigner: KeyringPair;
+    let authority: KeyringPair;
+
+    const { keyring } = testData;
+
+    beforeAll(async () => {
+        api = (await creditcoinApi((global as any).CREDITCOIN_API_URL)).api;
+        if (globals.CREDITCOIN_EXECUTE_SETUP_AUTHORITY) {
+            sudoSigner = keyring.addFromUri('//Alice');
+            authority = keyring.addFromUri('//Auth');
+            await addAuthorityAsync(api, authority.address, sudoSigner);
+        }
+    });
+
+    afterAll(async () => {
+        await api.disconnect();
+    });
+
+    testIf((global as any).CREDITCOIN_EXECUTE_SETUP_AUTHORITY, 'fee is 0', async () => {
+        const accountId = authority.address;
+        const sudoCall = api.tx.sudo.sudo(api.tx.creditcoin.deleteAuthority(accountId));
+        const { partialFee } = await sudoCall.paymentInfo(sudoSigner, { nonce: -1 });
+        expect(partialFee.toBigInt()).toBeGreaterThan(POINT_01_CTC);
```

Comment:
I'm not sure I understand this. 

Is `partialFee` the fee paid for the sudo call? 

If yes does it include any possible fees for the `deleteAuthority` call ? 

Originally I wanted to assert that the fee for deleteAuthority is zero. I'm not sure what behavior we're asserting as-is.